react/ui-components/src/hoc/app-config-screen-builder/RenderConditionalField.js

```
@@ -0,0 +1,259 @@
+        },
+        {
+          moduleName: "HCM",
+          masterName: "ID_TYPE_OPTIONS_POPULATOR",
+          schemaCode: "HCM.ID_TYPE_OPTIONS_POPULATOR",
+        },
+        {
+          moduleName: "HCM",
+          masterName: "DELIVERY_COMMENT_OPTIONS_POPULATOR",
+          schemaCode: "HCM.DELIVERY_COMMENT_OPTIONS_POPULATOR",
+        },
+      ];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)